packages/neuron-ui/src/tests/getSyncLeftTime/index.test.ts

```diff
@@ -2,10 +2,12 @@ import getSyncLeftTime from 'utils/getSyncLeftTime'
 import fixtures from './fixtures.json'
 
 describe('Test getCompensatedTime', () => {
-  const fixtureTable = Object.entries(fixtures).map(([title, { estimate, expected }]) => [title, estimate, expected])
+  const fixtureTable: [string, number | null, string][] = Object.entries(
+    fixtures
+  ).map(([title, { estimate, expected }]) => [title, estimate, expected])
 
   test.each(fixtureTable)(`%s`, (_title, estimate, expected) => {
     expect.assertions(1)
-    expect(getSyncLeftTime(estimate)).toBe(expected)
+    expect(getSyncLeftTime(estimate ?? undefined)).toBe(expected)
```

Comment:
We can't specify the type at the appointed index. The `estimate` type may be `string`.

Reply:
I added this because of type error in tests

```
export const getSyncLeftTime = (estimate: number | undefined) => {
```

`getSyncLeftTime` only accepts number | undefined